modules/openshift-cluster-limits.adoc

```diff
@@ -82,8 +82,8 @@ the system has enough CPU, memory, and disk to satisfy the application requireme
 
 |===
 
-In {product-title} {product-version}, half of a CPU core (500 millicore) is now reserved by
-the system compared to {product-title} 3.11 and previous versions.
+In {product-title} {product-version}, half of a CPU core (500 millicore) is
+reserved by the system compared to {product-title} 3.11 and previous versions.
 
-In {product-title} 4.1, the tested node limit has been lowered until scale tests can be run at
+In {product-title} {product-version}, the tested node limit has been lowered until scale tests can be run at
```

Comment:
This may go away based on tests happening right now.  Stay tuned...

Reply:
We can remove the caveat added around the number of nodes per cluster limit.